j2735-2024/src/test/java/j2735/SPAT/IntersectionStatusObjectTest.java

```diff
@@ -0,0 +1,111 @@
+package j2735.SPAT;
+
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Unit tests for the IntersectionStatusObject class.
+ * Generated using ChatGPT
+ */
+public class IntersectionStatusObjectTest {

```

Comment:
For bitstrings prefer testing the actual function they perform, which is to deserialize from text, rather than setting each bit programmatically and testing that it was set, which is very verbose for humans to try to review.  See the MapData LaneAttributes test for some examples.

Reply:
Addressed